cohort_banking_initiator_js/src/banking-app.ts

```diff
@@ -133,6 +133,22 @@ export class BankingApp {
                 writeset: [tx.from, tx.to],
                 readvers: state.items.map(i => i.version),
                 statemaps: [{ "TRANSFER": tx }],
+                onCommit: {
+                    publish: {
+                        kafka: [
+                            {
+                                topic: "test.transfer.feedback.js",
+                                value: {
+                                    "from_account": tx.from,
+                                    "to_account": tx.to,
+                                    "amount": tx.amount
+                                }
+                            },
+                        ]
+                    }
+                }
+
+                // TODO: GK - Add on commit action
```

Comment:
Is this supported? If we support action why dont we add them to interface too?

Reply:
This is a good point, messenger is written in a way that it can accept any valid json, but will only process the supported types. 

But on the other end, it would be nice if the sender is made aware on what they can send. Let me think on it and see how I can add this in at the cohort end. Thank for pointing this out!